style.css

```diff
@@ -391,9 +391,11 @@ input[type=text] {
   width: 50%;
   height: 4.5rem;
   margin: 1.5% 0;
+  background-color: #fff;
 }
 
```

Comment:
- [ ] [OPTIONAL] Please use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. Also, make sure to get the styling correct not just the raw text, as you can clearly see the difference between your readme and the template readme.